pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -253,7 +256,7 @@ func TestOutboundListenerConfig_WithSidecar(t *testing.T) {
 			&model.Port{
 				Name:     "udp",
```

Comment:
udp ?

Reply:
I just used the existing port and changed PROTOCOL. name really does not matter here I think. Do you want me to change?